Strategy11/formidable-forms
# Introducing Styling for Dropdown Placeholder Text

This PR introduces a new feature that enhances the visual appeal of our dropdown fields by adding styling to the placeholder text.

## Related Issues:
https://github.com/Strategy11/formidable-pro/issues/3300
https://github.com/Strategy11/formidable-pro/issues/4247

## QA URL:
https://qa.formidableforms.com/sherv4/wp-admin/admin.php?page=formidable-styles&frm_action=edit&id=5&form=20

## Testing Instructions:
1. Navigate to `WP Admin > Formidable`.
2. Open an existing form or create a new one.
3. Check the Dropdown field and ensure that the placeholder color matches the color set in other input placeholders. This can be verified under `Style Settings > Field Colors > Read Only Style`.

## Visual Outputs:
### Admin
![2023-06-01 15 45 08](https://github.com/Strategy11/formidable-forms/assets/69119241/95dc306a-4067-4c84-961a-6bf2b1e0b633)

## Frontend
![2023-06-01 15 45 57](https://github.com/Strategy11/formidable-forms/assets/69119241/e6c086f3-db89-4614-a198-2aabb6a88d35)
